onnxruntime/contrib_ops/cpu/bert/attention.cc

```diff
@@ -38,15 +39,25 @@ class Attention : public OpKernel, public AttentionCPUBase {
                                    int input_idx,
                                    /*out*/ bool& used_shared_buffers) override;
 
+  virtual std::optional<Tensor> GetPrePackTensor(int /*input_index*/) override;
+
+  Status SetPrePackTensor(int input_idx, const Tensor& pre_packed_tensor) override;
+
  private:
   bool IsPackWeightsSuccessful(int qkv_index, AllocatorPtr alloc, size_t head_size,
                                size_t input_hidden_size, const T* weights_data,
                                size_t weight_matrix_col_size, PrePackedWeights* prepacked_weights);
 
+  void ConvertPrePackWeightsIntoTensor(onnxruntime::AllocatorPtr& alloc, const onnxruntime::Tensor& weights, PrePackedWeights* prepacked_weights);
```

Comment:
Going by the name prepacked_weights can not be optional parameter otherwise there is nothing to prepack, wen can it be nullptr?
Is it an input or output?